apps/api-extractor/src/extractor/api-extractor.schema.json

```diff
@@ -8,6 +8,11 @@
       "type": "string"
     },
 
+    "extends": {
+      "description": "Optional field. Path to json config file from which config should be extended",
```

Comment:
>Optional field. Path to json config file from which config should be extended [](start = 22, length = 77)

`Optional field. Path to json config file from which config should extend`

Reply:
Fixed text